standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1473,9 +1473,11 @@ struct InsertEventRequestData {
 }
 
 union FireEventRequestData {
-    1: InsertEventRequestData insertData
```

Comment:
nit: we need to update [hive_metastore.proto]( https://github.com/apache/hive/blob/master/standalone-metastore/metastore-common/src/main/protobuf/org/apache/hadoop/hive/metastore/hive_metastore.proto) as well...
https://cwiki.apache.org/confluence/display/Hive/HiveDeveloperFAQ#HiveDeveloperFAQ-Howtogenerateprotobufcode?

Reply:
ack